### PR TITLE
Changed:variable default_sections to base_section #fixes 3395

### PR DESCRIPTION
**Pull request by ankit01ojha**
### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by namanyadav12** [comment]
Please follow coala.io/commit for writing the commit message.

**Comment by ankit01ojha** [comment]
@namanyadav12 ok, I'll keep that in mind. Thanks :)

File: coalib/settings/ConfigurationGathering.py [hunk 1]
```diff
@@ -163,15 +163,15 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     if bool(cli_sections['default'].get('no_config', 'False')):
         sections = cli_sections
     else:
-        default_sections = load_config_file(Constants.system_coafile,
-                                            log_printer)
+        base_sections = load_config_file(Constants.system_coafile,
+                                         log_printer)
```
Comment (vijeth-aradhya) [inline coalib/settings/ConfigurationGathering.py]: Also, I think you can put this in one line ( it will be < 80? )

**Comment by vijeth-aradhya** [comment]
The commit message should be in 'imperative mood', i.e 'Changed' could be 'Change..'

**Comment by ankit01ojha** [comment]
@vijeth-aradhya and @Nosferatul I have closed this pull request as it looks pretty messed up, so , can you both please review this new pull request https://github.com/coala/coala/pull/3438
Thanks :)

**Pull request closed without merging**